tests/lib/rules/import-align.js

```diff
@@ -25,21 +25,71 @@ ruleTester.run("import-align", rule, {
 
     valid: [
 
-        { code: "import foo from 'foo';\nimport bar from 'bar';\n", parserOptions: { sourceType: "module" } },
-        { code: "import foo                                from 'foo';\nimport supercalifragilisticexpialidocious from 'supercalifragilisticexpialidocious';\n", parserOptions: { sourceType: "module" } },
-        { code: "import foo                                from 'foo';\nimport supercalifragilisticexpialidocious from 'supercalifragilisticexpialidocious';\nimport {\n    A,\n    B\n} from 'foo';\n", parserOptions: { sourceType: "module" } }
+        {
+            code: "import foo from 'foo';\nimport bar from 'bar';\n",
+            parserOptions: { sourceType: "module" }
+        },
+        {
+            code: "import foo                                from 'foo';\nimport supercalifragilisticexpialidocious from 'supercalifragilisticexpialidocious';\n",
+            parserOptions: { sourceType: "module" }
+        },
+        {
+            code: "import foo                                from 'foo';\nimport supercalifragilisticexpialidocious from 'supercalifragilisticexpialidocious';\nimport {\n    A,\n    B\n} from 'foo';\n",
+            parserOptions: { sourceType: "module" }
+        }
```

Comment:
The content of existing tests didn't change, I just reformatted them for legibility, hope that's okay.